### PR TITLE
Travis ec2

### PR DESCRIPTION
**Pull request by dlukman**
Enables Travis to run test kitchen on EC2
@aboten

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,7 +1,32 @@
 require 'rspec/core/rake_task'
 require 'foodcritic'
+require 'kitchen'
 
 FoodCritic::Rake::LintTask.new
 RSpec::Core::RakeTask.new(:rspec)
 
-task default: [:foodcritic, :rspec]
+desc "Run kitchen tests sequentially"
```
Comment (kamaradclimber) [inline Rakefile]: OPT: factorize sequential and concurrent code using an concurrency enviroment variable.

File: Gemfile [hunk 2]
```diff
@@ -6,4 +6,9 @@ gem 'fauxhai',    '>= 2.2'
 gem 'foodcritic', '>= 4.0'
 gem 'rake'
 
+gem 'kitchen'
```
Comment (rveznaver) [inline Gemfile]: Could you group gems? http://bundler.io/groups.html

File: .kitchen.ec2.yml [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
 driver:
   name: ec2
-  instance_type: t2.micro # Free tier instance
+  instance_type: t2.micro
   associate_public_ip: true
-  region: <%= @region %>
-  availability_zone: <%= @availability_zone %>
-  security_group_ids: [<%= @group_id %>]
-  subnet_id: <%= @subnet_id %>
+  region: <%= ENV['AWS_REGION'] %>
```
Comment (rveznaver) [inline .kitchen.ec2.yml]:
``` ruby
<%= ENV['AWS_REGION'] || 'us-west-2' %>
```

File: Rakefile [hunk 4]
```diff
@@ -1,7 +1,36 @@
 require 'rspec/core/rake_task'
 require 'foodcritic'
+require 'kitchen'
 
 FoodCritic::Rake::LintTask.new
 RSpec::Core::RakeTask.new(:rspec)
 
-task default: [:foodcritic, :rspec]
+desc 'Run kitchen tests sequentially'
+task :ec2_sequential do
```
Comment (rveznaver) [inline Rakefile]: Is there a way to avoid code duplication between `:ec2_sequential` and `:ec2_concurrent`?

File: Gemfile [hunk 5]
```diff
@@ -5,5 +5,12 @@ gem 'chefspec',   '>= 4.2'
 gem 'fauxhai',    '>= 2.2'
 gem 'foodcritic', '>= 4.0'
 gem 'rake'
-
 gem 'keepass-password-generator'
+
+group :ec2 do
+	gem 'kitchen'
```
Comment (rveznaver) [inline Gemfile]: Spaces instead of tabs.

File: .kitchen.yml [hunk 6]
```diff
@@ -1,39 +1,42 @@
...
+      box: bento/centos-7.1
+  - name: windows-2012
+    driver_config:
+      box: opentable/win-2012r2-standard-amd64-nocm
 
 suites:
-- name: default
-  run_list:
-  - recipe[rundeck-node]
+  - name: default
```
Comment (rveznaver) [inline .kitchen.yml]: Seems indented one level too much (there should be no diff with the previous 3 lines).

**Comment by rveznaver** [comment]
+1

**Pull request merged**